packages/itwin/property-grid/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#----------------------------------------------------------------------------------------------
+# Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+# Licensed under the MIT License. See LICENSE.md in the project root for license terms.
+#----------------------------------------------------------------------------------------------
+
+FROM docker.io/library/debian:11
+
+# Downloads needed for git, npm and playwright to work
+RUN apt-get update \
+  && apt-get install -y curl \
+  && apt-get install -y libnss3 libatk-bridge2.0-0 libasound2 \
+  libcairo2 libcups2 libdbus-1-3 libatk1.0-0 libdrm2 libxrandr2 \
+  libglib2.0-0 libxkbcommon0 libnspr4 libatspi2.0-0 libgbm1\
+  libpango-1.0-0 libxcomposite1  libxdamage1  libxfixes3\
+  && curl -sL https://deb.nodesource.com/setup_18.x | bash - \
+  && apt-get install -y git nodejs locales zsh procps
```

Comment:
do we really need all these deps?

Reply:
The first part looks okay.

For the second part, `RUN npx playwright install-deps` definitely looks more future-proof than installing specific dependencies that may change with future releases of `playwright`. What was the failure?

Reply:
Have you looked at using a docker image made specially for playwright - https://playwright.dev/docs/docker?

Reply:
Changed "FROM docker.io/library/debian:11" to "FROM mcr.microsoft.com/playwright:v1.36.2-jammy"